javascript/src/main/App.js

```diff
@@ -28,6 +28,7 @@ import SearchHistory from "main/pages/SearchHistory/SearchHistory";
 import SearchResults from "./pages/Messages/SearchResults";
 import SearchedTerms from "main/pages/Admin/SearchedTerms";
 import ChannelPageLinks from "./pages/Channels/ChannelPageLinks";
+import Alluser from "main/pages/SearchHistory/Alluser";
```

Comment:
Can we keep our existing naming convention where we capitalize the first letter of each word? It looks like you forgot to capitalize the `U` in `user`.

Reply:
Fix format.